src/sdk/exchange/classes/cross-chain-bridge-route-transaction.ts

```diff
@@ -0,0 +1,85 @@
+import { BaseClass } from '../../common';
+import { IsNumber } from 'class-validator';
+
+export class ApprovalData {
+  minimumApprovalAmount: string;
+  approvalTokenAddress: string;
+  allowanceTarget: string;
+  owner: string;
+}
+
+export class CrossChainBridgeBridgeAsset {
+  chainId: number;
+  address: string;
+  symbol: string;
+  name: string;
+  decimals: number;
+  icon: string;
+  chainAgnosticId?: string;
+  rank: number;
+  updatedAt: string;
+  isEnabled: boolean;
+  createdAt: string;
+}
+
+export class CrossChainBridgeGasFee {
+  @IsNumber()
+  gasLimit: number;
+  asset: CrossChainBridgeBridgeAsset;
+  feesInUsd: number;
+}
+
+export class CrossChainBridgeProtocolFees {
+  @IsNumber()
+  amount: number;
+  feesInUsd: number;
+  asset: CrossChainBridgeBridgeAsset;
+}
+
+export class Protocol extends BaseClass<Protocol> {
+  name: string;
+  displayName: string;
+  icon: string;
+}
+
+export class Step extends BaseClass<Step> {
+  type: string;
+  chainId?: number;
+  fromChainId?: number;
+  fromAmount: string;
+  toAmount: string;
+  protocolFees: CrossChainBridgeProtocolFees;
```

Comment:
Missing `@Type(() => CrossChainBridgeProtocolFees)` ?

(same for the other properties)